scripts/tracing.js

```diff
@@ -1,41 +1,42 @@
 'use strict';
 
-const { propagation } = require('@opentelemetry/api');
+const { NodeSDK } = require('@opentelemetry/sdk-node');
 const { CompositePropagator, W3CBaggagePropagator, W3CTraceContextPropagator } = require('@opentelemetry/core');
-const { registerInstrumentations } = require('@opentelemetry/instrumentation');
 const { getNodeAutoInstrumentations } = require('@opentelemetry/auto-instrumentations-node');
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
+const { OTLPTraceExporter } = require('@opentelemetry/exporter-trace-otlp-grpc');
+const { OTLPMetricExporter } = require('@opentelemetry/exporter-metrics-otlp-grpc');
+const { PeriodicExportingMetricReader } = require('@opentelemetry/sdk-metrics');
 
 /**
  * Load directly from the dist folder because `tracing.js` is used outside normal NX
- * operations. That means the `openfeature-propagator` is not automatically bunded
+ * operations. That means the `openfeature-propagator` is not automatically bundled
  * during the build process.
  */
 const { OpenFeaturePropagator } = require('../dist/packages/openfeature-propagator/src/index');
 
-registerInstrumentations({
-  instrumentations: [getNodeAutoInstrumentations()],
-});
-
-const { Resource } = require('@opentelemetry/resources');
-const { SemanticResourceAttributes } = require('@opentelemetry/semantic-conventions');
-const { NodeTracerProvider } = require('@opentelemetry/sdk-trace-node');
-const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
-const { SimpleSpanProcessor } = require('@opentelemetry/sdk-trace-base');
-
-propagation.setGlobalPropagator(
-  new CompositePropagator({
-    propagators: [new W3CBaggagePropagator(), new W3CTraceContextPropagator(), new OpenFeaturePropagator()],
-  })
-);
-
 const serviceName = process.env['OTEL_SERVICE_NAME'] || 'fib3r';
+const collectorUrl = process.env['OTEL_COLLECTOR_URL'] || 'http://otel-collector:4317';
 
-const provider = new NodeTracerProvider({
+const sdk = new NodeSDK({
   resource: new Resource({
     [SemanticResourceAttributes.SERVICE_NAME]: serviceName,
   }),
+  instrumentations: [getNodeAutoInstrumentations()],
```

Comment:
Related - https://github.com/open-feature/flagd/issues/754

Unfortunately this is the reason for the error logs